lib/src/mistralai_client_dart_base.dart

```diff
@@ -103,6 +103,54 @@ class MistralAIClient {
     }
   }
 
+  Stream<ChatCompletionChunk> streamChat(ChatParams params) async* {
```

Comment:
There is no documentation, of this method. We have an [issue with documenting the public API](https://github.com/nomtek/mistralai_client_dart/issues/10)  but maybe it is worth adding even a simple doc now because we have some documentation in other methods already.

Reply:
Added in https://github.com/nomtek/mistralai_client_dart/pull/17/commits/dd58c82113acb49c5a3ec4dd702157bd201768ca